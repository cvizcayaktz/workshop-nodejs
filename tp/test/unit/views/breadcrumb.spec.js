const React = require('react');
const renderer = require('react-test-renderer');
const Breadcrumb = require('../../../views/components/Breadcrumb');

describe('Breadcrumb (Snapshot)', () => {
  const component = renderer.create(<Breadcrumb query="item" breadcrumbList={["cat1","cat2","cat3"]}/>);

  it('Breadcrumb renders', () => {
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
