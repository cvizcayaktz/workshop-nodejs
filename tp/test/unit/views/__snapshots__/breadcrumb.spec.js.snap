// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb (Snapshot) Breadcrumb renders 1`] = `
<ul
  className="breadcrumb"
>
  <li
    className="breadcrumb__element"
  >
    Inicio
  </li>
  <li
    className="breadcrumb__element"
  />
  <li
    className="breadcrumb__element"
  />
  <li
    className="breadcrumb__element"
  />
  <li
    className="breadcrumb__element"
  >
    "
    item
    "
  </li>
</ul>
`;
